syntax.php

```diff
@@ -119,13 +120,17 @@ function slackinvite_signinform() {
         $form->addElement(form_makeTextField('first_name', '', $this->getlang('first_name'), 'first__name'));
         $form->addElement(form_makeTextField('last_name', '', $this->getlang('last_name'), 'last__name'));
         $form->addElement(form_makeTextField('email', '', $this->getlang('email'), 'email'));
+        $captcha = $form->addElement(form_makeButton('Captcha'))
+        // Modify Captcha
+        $captcha->attr('class', 'g-recaptcha')
+        $captcha->attr('data-sitekey',"YOUR CAPTCHA TOKEN HERE")
```

Comment:
All of this fails - for starters, PHP sentences needs to be semicolon terminated (;) - otherwise you get syntax errors everywhere.